doc/port_fec_ber/port_fec_ber.md

```diff
@@ -0,0 +1,196 @@
+# Sonic Port FEC BER #
+
+## Table of Content 
+- [Revision](#revision)
+- [Scope](#scope)
+- [Definitions/Abbreviation](#abbreviations)
+- [1 Overview](#1-overview)
+  - [1.1 Functional requirements](#11-functional-requirements)
+  - [1.2 CLI requirements](#12-cli-requirements)
+- [2 Architecture Design](#2-architecture-design)
+- [3 High level design](#3-high-level-design)
+  - [3.1 Assumptions](#31-assumptions)
+  - [3.2 SAI counters used](#32-sai-counters-used)
+  - [3.3 SAI API](#33-sai-api)
+  - [3.4 Calculation formulas](#34-calculation-formulas)
+- [4 Sample output](#4-sample-output)
+
+
+ 
+### Revision  
+
+  | Rev |     Date    |       Author           | Change Description                |
+  |:---:|:-----------:|:----------------------:|-----------------------------------|
+  | 0.1 |             | Vincent (Ping Ching) Ng| Initial version                   |
+
+### Scope  
+
+  This document provide information about the implementation of Port Forward Error Correction (FEC) Bit Error Rate (BER) measurement. 
+  This new statistic will include correctable bits and uncorrectable bits BER.
+
+### Abbreviations 
+
+  FEC        - Forward Error Correction.  
+  BER        - Bits Error Rate, measure in bit per second.  
```

Comment:
The bit error rate (BER) is the number of bit errors per unit time. The bit error ratio (also BER) is the number of bit errors divided by the total number of transferred bits during a studied time interval. Bit error ratio is a unitless performance measure, often expressed as a [percentage](https://en.wikipedia.org/wiki/Percentage).[[1]](https://en.wikipedia.org/wiki/Bit_error_rate#cite_note-1)

Reply:
I think we are calculating bit error ration as a unitless number as we are calculating num error bits / total number tranfered. The calculation is not num error bits/sec